src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -144,3 +153,12 @@ def removeModule(pomFile, module) {
         w << pomContent
     }
 }
+
+def getLatestSDK(archetypeVersion) {
```

Comment:
`archetypeVersion` is currently not used as the child element `latest` is used.

Reply:
See https://github.com/adobe/aem-project-archetype/pull/334#discussion_r377198238